src/ui.css

```diff
@@ -60,3 +60,33 @@
   padding: 0 1em;
   line-height: 0;
 }
+
+.ytaf-remote-button {
+  display: inline-block;
+  width: 21.34px;
+  height: 21.34px;
```

Comment:
Where did you get `21.34px`?

Reply:
I think it was the computed height of the text before button. And I wanted to keep the height the button same as text, so it doesn't look weird.

Reply:
Hmm. I think it must have originally been 16pt (21.3333...px).